css/badge.css

```diff
@@ -0,0 +1,99 @@
+:root{
+    --primary-color:#007BFF;
+    --success-color:#28A745;
+    --error-color:#DC3545;
+    --info-color:#17A2B8;
+    --warning-color: #FFC107;
+}
+
+.error-badge,.primary-badge,.success-badge,.info-badge,.warning-badge{
+    width: fit-content;
+    padding: 0.2rem;
+    border-radius: 6px;
+    margin: 0.1rem;
+    font-size: 14px;
+}
+.error-badge{
+    background-color: var(--error-color);
+    color: #fff;
+}
+.primary-badge{
+    color: #fff;
+    background-color: var(--primary-color);
+}
+.success-badge{
+    color: #fff;
+    background-color: var(--success-color);
+}
+.info-badge{
+    color: #fff;
+    background-color: var(--info-color);
+}
+.warning-badge{
+    color: #fff;
+    background-color: var(--warning-color);
```

Comment:
My suggestion is you can create a new .badge class. The common style element **color: #fff;** can be placed in .badge class. You will need to update your HTML code as in example below:
`<span class="badge info-badge"> Info </span>`

Also, colors should be placed in root element. Let me know if you have different thoughts.

Reply:
Done with the changes suggested by you.